run_project_tests.py

```diff
@@ -122,6 +123,8 @@ def add_step(self, step: BuildStep, stdo: str, stde: str, mlog: str = '', time:
     def fail(self, msg: str) -> None:
         self.msg = msg
 
+python_info: 'PythonIntrospectionDict'
```

Comment:
why can't this just be assigned here at module time, instead of with the global hack in the InstallFile initializer?

Reply:
Hmm... I wasn't sure whether it should run at script load or only once used. In retrospect, it is still used every single time test.json is parsed, not just for python, so... I guess we could do this globally.